### PR TITLE
Estonian-6.17.0-2

### PR DESCRIPTION
**Pull request by ToomasMolder**

### CODE REVIEW

**Comment by iluwatar** [comment]
Pull request name and description are not following the workflow. See workflow chapter 9.1 https://github.com/vrk-kpa/xroad-joint-development/blob/master/WORKFLOW.md#9-pull-requests

File: doc/Architecture/arc-g_x-road_arhitecture.md [hunk 1]
```diff
@@ -1,10 +1,10 @@
...
-
----
 
+| ![European Union / European Regional Development Fund / Investing in your future](img/eu_rdf_75_en.png "Documents that are tagged with EU/SF logos must keep the logos until 1.1.2022, if it has not stated otherwise in the documentation. If new documentation is created  using EU/SF resources the logos must be tagged appropriately so that the deadline for logos could be found.") |
+| -------------------------: |
 
 # Technical Specification
-**X-Road Architecture**
+
+# X-Road Architecture
```
Comment (VitaliStupin) [inline doc/Architecture/arc-g_x-road_arhitecture.md]: Should be:
# X-Road Architecture
**Technical Specification**

File: doc/Manuals/ig-ss_x-road_v6_security_server_installation_guide.md [hunk 2]
```diff
@@ -1,10 +1,10 @@
...
-
----
 
+| ![European Union / European Regional Development Fund / Investing in your future](img/eu_rdf_75_en.png "Documents that are tagged with EU/SF logos must keep the logos until 1.1.2022, if it has not stated otherwise in the documentation. If new documentation is created  using EU/SF resources the logos must be tagged appropriately so that the deadline for logos could be found.") |
+| -------------------------: |
 
 # Security Server Installation Guide
-**X-ROAD 6**
+
+# X-ROAD 6
```
Comment (VitaliStupin) [inline doc/Manuals/ig-ss_x-road_v6_security_server_installation_guide.md]: No need for double header. Could be "X-ROAD 6 Security Server Installation Guide" instead, or could be left unchanged

File: doc/Architecture/arc-ss_x-road_security_server_architecture.md [hunk 3]
```diff
@@ -1,10 +1,10 @@
...
-
----
 
+| ![European Union / European Regional Development Fund / Investing in your future](img/eu_rdf_75_en.png "Documents that are tagged with EU/SF logos must keep the logos until 1.1.2022, if it has not stated otherwise in the documentation. If new documentation is created  using EU/SF resources the logos must be tagged appropriately so that the deadline for logos could be found.") |
+| -------------------------: |
 
 # X-Road: Security Server Architecture
-**Technical Specification**
+
+# Technical Specification
```
Comment (VitaliStupin) [inline doc/Architecture/arc-ss_x-road_security_server_architecture.md]: "Technical Specification" should not be header.

File: doc/Manuals/ug-ss_x-road_6_security_server_user_guide.md [hunk 4]
```diff
@@ -1,10 +1,10 @@
...
-
----
 
+| ![European Union / European Regional Development Fund / Investing in your future](img/eu_rdf_75_en.png "Documents that are tagged with EU/SF logos must keep the logos until 1.1.2022, if it has not stated otherwise in the documentation. If new documentation is created  using EU/SF resources the logos must be tagged appropriately so that the deadline for logos could be found.") |
+| -------------------------: |
 
 # SECURITY SERVER USER GUIDE
-**X-ROAD 6**
+
+# X-ROAD 6
```
Comment (VitaliStupin) [inline doc/Manuals/ug-ss_x-road_6_security_server_user_guide.md]: double header

File: doc/OperationalMonitoring/Protocols/pr-opmonjmx_x-road_operational_monitoring_jmx_protocol_Y-1096-3.md [hunk 5]
```diff
@@ -1,6 +1,10 @@
+
+| ![European Union / European Regional Development Fund / Investing in your future](../img/eu_rdf_75_en.png "Documents that are tagged with EU/SF logos must keep the logos until 1.1.2022, if it has not stated otherwise in the documentation. If new documentation is created  using EU/SF resources the logos must be tagged appropriately so that the deadline for logos could be found.") |
+| -------------------------: |
+
 # X-Road: Operational Monitoring JMX Protocol
 
-Technical Specification
+# Technical Specification
```
Comment (VitaliStupin) [inline doc/OperationalMonitoring/Protocols/pr-opmonjmx_x-road_operational_monitoring_jmx_protocol_Y-1096-3.md]: double header

File: doc/OperationalMonitoring/Protocols/pr-opmon_x-road_operational_monitoring_protocol_Y-1096-2.md [hunk 6]
```diff
@@ -1,6 +1,10 @@
+
+| ![European Union / European Regional Development Fund / Investing in your future](../img/eu_rdf_75_en.png "Documents that are tagged with EU/SF logos must keep the logos until 1.1.2022, if it has not stated otherwise in the documentation. If new documentation is created  using EU/SF resources the logos must be tagged appropriately so that the deadline for logos could be found.") |
+| -------------------------: |
+
 # X-Road: Operational Monitoring Protocol
 
-**Technical Specification**
+# Technical Specification
```
Comment (VitaliStupin) [inline doc/OperationalMonitoring/Protocols/pr-opmon_x-road_operational_monitoring_protocol_Y-1096-2.md]: double header

File: doc/UseCases/uc-mess_x-road_member_communication_use_case_model.md [hunk 7]
```diff
@@ -1,10 +1,10 @@
...
-
----
 
+| ![European Union / European Regional Development Fund / Investing in your future](img/eu_rdf_75_en.png "Documents that are tagged with EU/SF logos must keep the logos until 1.1.2022, if it has not stated otherwise in the documentation. If new documentation is created  using EU/SF resources the logos must be tagged appropriately so that the deadline for logos could be found.") |
+| -------------------------: |
 
 # X-Road: Use Case Model for Member Communication
-**Analysis**
+
+# Analysis
```
Comment (VitaliStupin) [inline doc/UseCases/uc-mess_x-road_member_communication_use_case_model.md]: double header

**Review by VitaliStupin — CHANGES_REQUESTED** [review]

**Comment by ToomasMolder** [comment]
Estonian-6.17.0-2: EU RDF SF logos attached

**Review by VitaliStupin — APPROVED** [review]

**Review by iluwatar — APPROVED** [review]

**Pull request merged**